qemu-runner.sh

```diff
@@ -4,6 +4,8 @@ set -e
 ELF="$1"
 BIN="${ELF%.elf}.bin"
 
+sudo bash -c "./mkrootfs-aarch64.sh"
```

Comment:
I don't think running a command via `sudo` which is invoked after a `cargo build` is great from a security standpoint.

Maybe it would be better to check whether the initrd exists and, if not, print out a helpful error message suggesting to run the creation script instead? That way users aren't suprised that `sudo` is asking for a password after running `cargo run`.